graphprot/NeuralNet.py

```diff
@@ -372,6 +374,7 @@ def test(self, database_test, threshold=4, hdf5='test_data.hdf5'):
 
         self.test_out = _out
         self.test_y = _y
+        print(_out, _y)
```

Comment:
maybe remove the print :) With the code being serious now we should actually think of a logging system